packages/wonder-blocks-button/src/components/button-core.js

```diff
@@ -327,9 +332,18 @@ const _generateStyles = (color, kind, light, iconWidth, size) => {
                 color: light ? white50 : offBlack32,
                 borderColor: light ? fadedColor : offBlack32,
                 cursor: "default",
+                ":focus": {
+                    borderColor: light ? white50 : offBlack32,
+                    borderWidth: 2,
+                    // The left padding for the button with icon should have 4px
+                    // less padding
+                    paddingLeft: iconWidth ? padding - 5 : padding - 1,
```

Comment:
Nit: It would be probably good to include a comment explaining why this differs from the `default` state. Also, it would probably be helpful to move these values into a constant so it's easier to keep track of those subtle changes (-5, -1, etc).